### DISCUSSION TITLE
Pade via url

### DISCUSSION THREAD

**Discussion by davorgegorincic** [igniterealtime/Pade#264] [Q&A]
Hello! I wonder how to set the pade so that the user can access directly via url and not by extension. I can access via chrome extensions, but i can't access it via url.

**Comment by deleolajide** [comment]
https://igniterealtime.github.io/pade/

**Comment by deleolajide** [comment]
Please note that features that depend on the browser extension permissions like RSS feeds will not work. Also you would need to set your connection parameters and Jitsi meeting connection parameters to the appropriate server
